### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by robertolopezlopez**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (mgwoj) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: In our case it is Linux, MacOS, etc.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (mgwoj) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Please remove it

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (mgwoj) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: We need TF version rather

**Review by mgwoj — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (robertolopezlopez) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: I thought I had already... weird

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (robertolopezlopez) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: same here, I thought I had changed it already with the web editor

Comment (robertolopezlopez) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: In any case, this is httpie. Terraform has nothing to do here

**Review by mgwoj — APPROVED** [review]

**Pull request merged**